SnapchatNavigation/Navigation/ButtonsController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  ButtonsController.swift
+//  SnapchatNavigation
+//
+//  Created by Artur Chabera on 08/03/2019.
+//  Copyright © 2019 Artur Chabera. All rights reserved.
+//
+
+import UIKit
+
+protocol ButtonsDelegate: class {
+    func scroll(to panel: Panel)
+    func backToCamera()
+}
+
+class ButtonsController: UIViewController {
+
+    weak var delegate: ButtonsDelegate?
+
+    private var leftButton: UIButton!
```

Comment:
Instead of unwrapped optional, we can use:
private let leftButton = UIButton.make(.left)